src/components/HermesSkillsStore.tsx

```
@@ -358,42 +384,210 @@ export default function HermesSkillsStore({ testId }: { testId?: string }) {
-    const opts = catalog.providers.map((p) => ({ id: p.id, label: `${p.id} (${p.count})` }));
-    if (catalog.provider && !opts.some((o) => o.id === catalog.provider)) {
-      opts.push({ id: catalog.provider, label: catalog.provider });
+  // The result count, announced politely — but only when the FILTERS moved.
+  // Announcing on every render would narrate scrolling, and announcing while a
+  // request is still in flight would read out the previous answer's total.
+  const resultCount = browsing ? catalog.total : installedFiltered.length;
+  const filterSignature = groups.map((g) => `${g.id}:${g.selected.join('+')}`).join('|');
+  const announced = useRef<string | null>(null);
+  useEffect(() => {
+    announced.current = null;
+    setFilterLive('');
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/HermesSkillsStore.tsx:579:5
  577 |   useEffect(() => {
  578 |     announced.current = null;
> 579 |     setFilterLive('');
      |     ^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  580 |   }, [tab]);
  581 |   useEffect(() => {
  582 |     if (browsing && (catalog.loading || catalog.stale)) return;

(react-hooks/set-state-in-effect)